src/bindings/python/flux/cli/plugin.py

```diff
@@ -0,0 +1,180 @@
+##############################################################
+# Copyright 2024 Lawrence Livermore National Security, LLC
```

Comment:
does this copyright year need to be updated to 2025?

Reply:
What we actually do is put the year the file was originally included in the source, then never worry about updating it. I don't have a problem with removing the year, but would like to get an ACK from @garlick since we haven't yet dropped the year everywhere else.

Reply:
Ok, that seems like maybe an out of scope conversation for this PR. I'll put 2025 in the header to be consistent and we can talk about dropping the year later.